assets/files.wxs

```diff
@@ -1930,6 +1930,11 @@
             <File Id="filFF877991123D49A58FD15CB83AD44BA7" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-windows`
Validate New-User.Can create New-User with explicit properties in local machine
```powershell
Cannot process command because of one or more missing mandatory parameters: Name Password.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 40
40:             $result = New-User -AccountStore Machine

```
Validate New-User.Errors on Name argument of empty string, null, spaces
```powershell
Expected an exception, with FullyQualifiedErrorId 'InvalidValue,Microsoft.PowerShell.Commands.NewUserCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 107
107:             { New-User -Name "   " -PasswordNotRequired } | Should -Throw -ErrorId "InvalidValue,Microsoft.PowerShell.Commands.NewUserCommand"
```
Validate New-User.Error on user exists
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParameterArgumentValidationError,Microsoft.PowerShell.Commands.NewUserCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 112
112:             $exc = { New-User -Name TestUserNew -PasswordNotRequired -AccountStore Machine } | Should -PassThru -Throw -ErrorId "ParameterArgumentValidationError,Microsoft.PowerShell.Commands.NewUserCommand"
```